tables/he.utb

```diff
@@ -66,21 +57,13 @@ word 'em =
 word 'tis =
 word 'twas =
 begnum # 456-1456 print number sign before number
-midnum , 2
-postpunc , 2
-word , 2
+always , 2
```

Comment:
OK. I presume the original author told you that comma should always be dots 2, and under no circumstance the `punctuation , 6` from en-chardefs.cti should be used. In that case, this change is correct. Otherwise, note that the `always , 2` is not equivalent to the three other rules. (But I assume you know.)